modules/services/utils/src/main/java/au/com/shiftyjelly/pocketcasts/utils/SharingUrlTimestampParser.kt

```diff
@@ -0,0 +1,37 @@
+package au.com.shiftyjelly.pocketcasts.utils
+
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.seconds
+
+class SharingUrlTimestampParser {
+    fun parseTimestamp(t: String): Pair<Duration?, Duration?> {
+        val intervalPattern = Regex("""(\d*\.?\d*)?,?(\d*\.?\d*)?""") // t=10,20 or t=,20 or t=10
```

Comment:
This needs to handle also decimal points as well. Number after the decimal point is considered to be milliseconds.